AppHandling/Sort-AppFoldersByDependencies.ps1

```diff
@@ -84,50 +98,72 @@ try {
     $script:unresolvedDependencies = $()
 
     function AddAnApp { Param($anApp) 
+        $rebuildThis = $false

```

Comment:
I don't get why this is called `$rebuildThis`. Nothing is being built around here.